mungegithub/mungers/approvers/owners.go

```diff
@@ -214,7 +215,18 @@ func (ap Approvers) GetFiles() []File {
 }
 
 func (ap Approvers) GetCCs() []string {
-	return ap.Owners.GetSuggestedApprovers().Difference(ap.Approvers).List()
+	approvers := []string{}
```

Comment:
This is more likely to return an empty list, we should make sure we don't print a sentence in the message with an empty list of suggested approvers. Will fix.